files/templates/alertmanager/alertmanager.yaml

```diff
@@ -54,7 +54,7 @@ route:
   # Team Atlas Slack (team)
   - receiver: team_atlas_slack
     matchers:
-    - severity="page"
+    - severity!="page"
```

Comment:
What happens for pages on testing installations?

Reply:
They are silenced by opsgenie

Reply:
Are we sure we want to restrict that? Alerts in the slack channel renders some useful links (runbook, dashboard)

Reply:
I mean:
- `notify` on `stable` clusters => slack #alert-atlas :heavy_check_mark: 
- `page` on `stable` clusters => Opsgenie :heavy_check_mark: 
- `notify` on `testing` clusters => slack #alert-atlas-test :heavy_check_mark: 
- `page` on `testign` clusters => Opsgenie and silenced = black hole. :negative_squared_cross_mark: 

I think we also want to see "page" alerts for testing clusters somewhere.

Reply:
How about page and testing go to #alert-atlas-test as well?

Reply:
Silenced alert in opsgenie are basically automatically closed, but you can still find them via using the following query : `tag: "testing"


Is this good enough ? Or do we want to send them to slack also ? This would involves a lot more config here.

Reply:
Should be good now ;)